src/rvoice/fluid_rvoice_mixer.c

```diff
@@ -736,7 +736,7 @@ fluid_mixer_get_mt_rvoice(fluid_rvoice_mixer_t* mixer)
 #define THREAD_BUF_TERMINATE 3
 
 /* Core thread function (processes voices in parallel to primary synthesis thread) */
-static void
+static FLUID_THREAD_RETURN_TYPE
```

Comment:
Why?

Reply:
the thread function return type might be different (for example with pthread it's void*)

Reply:
Then you should have setup a `typedef fluid_thread_return_t` and consistently applied that everywhere we use `new_fluid_thread` and also to `fluid_thread_func_t`.

Reply:
on it